app/build.gradle

```diff
@@ -17,11 +17,13 @@ android {
         debug {
             versionNameSuffix "Version: "
             resValue "string", "version", "${versionNameSuffix}${defaultConfig.versionName}"
+            resValue "string", "google_maps_key", "AIzaSyBFvrqI8_J108WntI7surDGqCUcd1RbHPQ"
```

Comment:
Refer this from a resource file

Reply:
I couldn't find proper way to access string resource in gradle file . So as suggested in this article https://medium.com/code-better/hiding-api-keys-from-your-android-repository-b23f5598b906 i am adding key to gradle.properties and then accessing it in build.gradle.